libs/remix-ui/home-tab/src/lib/components/homeTabGetStarted.tsx

```diff
@@ -30,31 +30,31 @@ const workspaceTemplates: WorkspaceTemplate[] = [
   {
     gsID: 'sUTLogo',
     workspaceTitle: 'Start Coding',
-    description: 'Create a new project using this template.',
+    description: 'Start coding using the default template.',
     projectLogo: 'assets/img/remixverticaltextLogo.png',
     templateName: 'remixDefault',
   },
   {
     gsID: 'sUTLogo',
-    workspaceTitle: 'Circom',
+    workspaceTitle: 'Zk Semaphore',
     description: 'Create a new ZK Project with Circom using this template.',
     projectLogo: 'assets/img/circom.webp',
     templateName: 'semaphore',
   },
-  {
-    gsID: 'sUTLogo',
-    workspaceTitle: 'Uniswap',
-    description: 'Create a new MultiSig wallet using this template.',
-    projectLogo: 'assets/img/gnosissafeLogo.png',
-    templateName: 'uniswapV4Template',
-  },
   {
     gsID: 'sUTLogo',
     workspaceTitle: 'ERC20',
     description: 'Create a new ERC20 token using this template.',
     projectLogo: 'assets/img/oxprojectLogo.png',
     templateName: 'ozerc20',
   },
+  {
+    gsID: 'sUTLogo',
+    workspaceTitle: 'Uniswap V4 Basic',
```

Comment:
I think this should be:
Uniswap v4 Hooks
because in the README.md in this template it says,"A template for writing Uniswap v4 Hooks"

Reply:
Done